### ISSUE TITLE
argc, argv in main

### ISSUE THREAD

**Issue by KimNaRyeong** [swtv-kaist/cs458-fall22#20]
main함수에서 입력 받아서 변수 개수가 맞지 않을 때 에러를 발생시키는 것과 입력 받은 값을 각 변수에 저장하는 부분을 지워도 되나요?

이부분을 지우면 잘 실행되는데 지우지 않으면 "run_crown ./tcas-mod-crown 50 -dfs" 했을 때 제대로 tc 생성이 안됩니다.

**Comment by moonzoo** [comment]
> main함수에서 입력 받아서 변수 개수가 맞지 않을 때 에러를 발생시키는 것과 입력 받은 값을 각 변수에 저장하는 부분을 지워도 되나요?

How to declare and handle symbolic variables is up to you if you have clear reasons to do it.

**Comment by KimNaRyeong** [comment]
감사합니다.

**Comment by KimNaRyeong** [comment]
감사합니다. 

혹시 실행커맨드가 "run_crown ./tcas-mod-crown 50 -dfs" 이게 맞나요? 아니면 ./tcas-mod-crown 뒤에 input을 더 붙여줘야 하나요? segmentation fault가 뜨는데 argv에 접근해서 그런 것 같아 보입니다.

**Comment by moonzoo** [comment]
Sorry that I misunderstood your previous question. I modified my previous comment.

**Comment by KimNaRyeong** [comment]
죄송하지만 아직 이해가 안되는데요..
기존의 코드를 추가하거나 삭제해도 되는건가요? 
저는 원래라면 argv 배열에 담겼을 사용자 입력값들을 심볼릭 변수로 쓰고 싶은데 이걸 새로 만든 배열에 저장하고 그 배열로부터 Cur_Vertical_Sep 같은 변수 값들을 저장하고 싶습니다. 그러려면 기존 코드를 좀 수정해야하는데 가능할까요?

**Comment by moonzoo** [comment]
> 기존의 코드를 추가하거나 삭제해도 되는건가요?

Yes, you can change the original code for symbolic variable setting.
For example,

```
#ifdef CROWN
SYM_int(A);
#else
A=atoi(argv[3]);
#endif
```
